packages/now-next/test/fixtures/05-spr-support/pages/api/noop.js

```diff
@@ -0,0 +1,3 @@
+export default (_, res) => {
```

Comment:
An API route shouldn't be needed for revalidation support right?

Reply:
It was specifically failing on the `/forever` route(s) that set revalidation to `false`. Our tests were asserting the `PRERENDER` header instead of `MISS` (new header).

Instead of updating the tests to check for `MISS` (which could let this regress into a lambda), I opted-into preview deployment so it used the Prerender type.

In the future, hits from L4 cache should respond different than `MISS` so we can appropriately detect it vs a lambda. This needs to be discussed with the CDN team.